dockerfile-generator/dockerfile-gen-services/src/main/java/module-info.java

```diff
@@ -1,36 +0,0 @@
-/**
- * Copyright 2019 Pramati Prism, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-module dockerfilegenservices {
-	exports io.hyscale.dockerfile.gen.services.generator;
```

Comment:
why is this removed ?

Reply:
Keeping module-info restricts how we write the test cases. We can have a discussion on this.